src/tests/stub_MemStore.cc

```diff
@@ -36,6 +36,7 @@ uint64_t MemStore::currentCount() const STUB_RETVAL(0)
 int64_t MemStore::maxObjectSize() const STUB_RETVAL(0)
 bool MemStore::dereference(StoreEntry &) STUB_RETVAL(false)
 void MemStore::markForUnlink(StoreEntry&) STUB
+void MemStore::unlinkByKeyIfFound(const cache_key *key) STUB
```

Comment:
Done.
BTW, there are currently lots of unused variables in Squid sources. I assume this should be fixed first before we are going to test new patches with 'strict' compilers.